src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <EnableWinRT>true</EnableWinRT>
```

Comment:
> I don't think i see where this property would enable such use though

Right, nothing attached to this property right now is related to WinRT use outside of UWP.